pyhms/tree.py

```diff
@@ -100,6 +101,12 @@ def all_individuals(self) -> list[Individual]:
             individuals_from_all_demes.extend(deme.all_individuals)
         return individuals_from_all_demes
 
+    @property
+    def solutions(self) -> list[Individual]:
```

Comment:
best individuals per leaf are often referred to as solutions. Maybe this property should be called specificaly r5s_solutions?